### PR TITLE
PBM-653 Created a contributing guide

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+## Submitting a pull request
+
+All bug reports, enhancements and feature requests are tracked in [Jira issue tracker](https://jira.percona.com/projects/PBM). Though not mandatory, we encourage you to first check for a bug report among Jira issues and in the PR list: perhaps the bug has already been addressed. 
+
+For feature requests and enhancements, we do ask you to create a Jira issue, describe your idea and discuss the design with us. This way we align your ideas with our vision for the product development.
+
+If the bug hasn’t been reported / addressed, or we’ve agreed on the enhancement implementation with you, do the following:
+
+1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this repository
+2. Clone this repository on your machine and sync it with upstream.
```
Comment (akira-kurogane) [inline CONTRIBUTING.md:43]:
```suggestion
2. Clone this repository on your machine
```

At the moment you do a clone it will be synchronized with the latest upstream commits.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+All bug reports, enhancements and feature requests are tracked in [Jira issue tracker](https://jira.percona.com/projects/PBM). Though not mandatory, we encourage you to first check for a bug report among Jira issues and in the PR list: perhaps the bug has already been addressed. 
+
+For feature requests and enhancements, we do ask you to create a Jira issue, describe your idea and discuss the design with us. This way we align your ideas with our vision for the product development.
+
+If the bug hasn’t been reported / addressed, or we’ve agreed on the enhancement implementation with you, do the following:
+
+1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this repository
+2. Clone this repository on your machine and sync it with upstream. 
+3. Create a branch for your changes based on the corresponding version branch. 
+4. Make your changes. Use the 80 characters line wrapping to improve code readability
```
Comment (akira-kurogane) [inline CONTRIBUTING.md:45]:
```suggestion
4. Make your changes. (Please use the 80 characters line wrapping to improve code readability)
```

**Review by akira-kurogane — APPROVED** [review]
I have two really minor edit suggestions. Pre-emptively approve the PR.

File: CONTRIBUTING.md (hunk 2 above)
Comment (dAdAbird) [reply CONTRIBUTING.md:45]: We shouldn't insist on 80 char lines.  It's hard to follow in Go and it isn't a common practice with Go code. And a lot of current PBM code doesn't fit into 80 lines.
Instead, I'd like to make some reference to this: https://github.com/golang/go/wiki/CodeReviewComments as the style guide to follow.

**Review by dAdAbird — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (nastena1606) [reply CONTRIBUTING.md]: Fixed

**Review by dAdAbird — APPROVED** [review]
